proc_nutation.py

```diff
@@ -49,10 +51,12 @@
     s.ft("t2")
     fl.next("After rx offset correction")
     fl.image(s)
+    # }}}
     if "amp" in s.dimlabels:
         plen = s.get_prop("acq_params")["p90_us"] * 1e-6
         logger.info(strm("pulse length is:", plen))
     s.ift("t2")
+    # {{{ PR why would you not just use the fid_from_echo here??
```

Comment:
deal with this (applies to entire bracketed region)

Reply:
again this was an old old version not in the examples repo and has since been deleted. The up to date version is inside the examples directory